plugin/trino-delta-lake/pom.xml

```diff
@@ -426,6 +432,15 @@
                             </ignoredResourcePatterns>
                         </configuration>
                     </plugin>
+
+                    <plugin>
+                        <artifactId>maven-dependency-plugin</artifactId>
+                        <configuration>
+                            <ignoredDependencies>
+                               com.amazonaws:aws-java-sdk-core
```

Comment:
we write these as `<dependency> ... `

Reply:
The newly added glue test uses `aws-java-sdk-core`.
That is why `maven-dependency-plugin` is complaining when not specifying the dependency as a compile dependency.
When I specify the dependency as compile dependency, it says it is test only.
When I specify it as test only it complains again.

Reply:
https://maven.apache.org/plugins/maven-dependency-plugin/examples/exclude-dependencies-from-dependency-analysis.html

Reply:
This falls into `<!-- used by tests but also needed transitively -->` category. I pushed a fix for this.